goaviatrix/firewall_instance.go

```diff
@@ -118,6 +118,12 @@ func (c *Client) GetFirewallInstance(firewallInstance *FirewallInstance) (*Firew
 		return nil, errors.New("Rest API get_instance_by_id Get failed: " + data.Reason)
 	}
 	if data.Results.InstanceID == firewallInstance.InstanceID {
+		if strings.HasPrefix(data.Results.KeyName, "vpc-") {
+			index := strings.Index(data.Results.KeyName, "_")
+			if index > -1 {
+				data.Results.FirewallName = data.Results.KeyName[index+1:]
```

Comment:
This logic is completely incorrect.  We cannot get instance name from key name. Key name is the name for the SSH key pait